src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -32,7 +37,7 @@ if (filterUsers === null) {
 
 const filterUsers = filterUsersRaw ? JSON.parse(filterUsersRaw) : [];
 const notificationFeedSrc =
-  "${REPL_ACCOUNT}/widget/NearOrg.Notifications.Notifications";
+  "${REPL_ACCOUNT}/widget/NearOrg.Notifications.NotificationsList";
```

Comment:
This is actually why notification counter got broken. In this [file](https://github.com/near/near-discovery-components/blob/develop/src/NearOrg/Notifications/NotificationsList.jsx#L21) we are making `Storage.set("lastBlockHeight", item.blockHeight)`. We've been trying to get a `lastBlockHeight` (line 41) from the given source component but that variable was in another component.